src/css/information.scss

```diff
@@ -15,112 +15,124 @@
   .quick-id {
     color: $white;
     background: $navy;
+    float: left;
 
-    h2,
-    table {
-      width: 310px;
-    }
-
-    h2 {
-      color: $white;
-    }
+    .content {
+      position: relative;
+      top: 10px;
+      left: 60%;
 
-    table {
-      line-height: 2;
-      font-weight: lighter;
-    }
+      h2,
+      table {
+        width: 310px;
+      }
 
-    a {
-      &,
-      &:hover {
+      h2 {
         color: $white;
-        text-decoration: none;
       }
-    }
 
-    .label {
-      width: 120px;
-      font-weight: bold;
+      table {
+        line-height: 2;
+        font-weight: lighter;
+      }
+
+      a {
+        &,
+        &:hover {
+          color: $white;
+          text-decoration: none;
+        }
+      }
+
+      .label {
+        width: 120px;
+        font-weight: bold;
+      }
     }
   }
 
   .contact-me {
     background: $cyan;
+    float: right;
+    .content {
+      position: relative;
+      top: 10px;
+
+      h2,
+      form {
+        width: 430px;
+      }
 
-    h2,
-    form {
-      width: 430px;
-    }
-
-    label {
-      @include text-hide();
-    }
-
-    input {
-      @include vertical-center(42px);
-      padding: 3px;
-      padding-left: 12px;
-    }
+      label {
+        @include text-hide();
+      }
 
-    textarea {
-      resize: none;
-      margin-bottom: 10px;
-      padding: 10px 12px;
-      height: 72px;
-    }
+      input {
+        @include vertical-center(42px);
+        padding: 3px;
+        padding-left: 12px;
+      }
 
-    input,
-    textarea {
-      width: 100%;
-      outline: none;
-      border: 1px solid transparent;
-      border-bottom-color: $grey;
-      background: transparent;
-      margin-bottom: 3px;
-
-      &::placeholder {
-        color: $grey;
+      textarea {
+        resize: none;
+        margin-bottom: 10px;
+        padding: 10px 12px;
+        height: 72px;
       }
 
-      &:focus {
-        background: $white;
-        border-color: $grey;
+      input,
+      textarea {
+        width: 100%;
+        outline: none;
+        border: 1px solid transparent;
+        border-bottom-color: $grey;
+        background: transparent;
+        margin-bottom: 3px;
+
+        &::placeholder {
+          color: $grey;
+        }
+
+        &:focus {
+          background: $white;
+          border-color: $grey;
+        }
       }
-    }
 
-    .two-column {
-      // https://css-tricks.com/fighting-the-space-between-inline-block-elements/#article-header-id-3
-      font-size: 0;
+      .two-column {
+        // https://css-tricks.com/fighting-the-space-between-inline-block-elements/#article-header-id-3
+        font-size: 0;
```

Comment:
I always think that using `display: inlne-block` is quite weird. 
Have you considered using flex or float here?